src/components/blocks/object-image-overlay/_object-image-overlay.js

```diff
@@ -304,6 +304,37 @@ const oicInit = () => {
             oic.advance(true);
           }
         };
+
+        oic.ontouchstart = (e4) => {
```

Comment:
Possibly consider updating older DOM Level 0 event handling to DOM Level 2 standardised method, i.e.,

```
oic.addEventListener('touchstart', (e4) => {
  const startXY = [e4.touches[0].pageX, e4.touches[0].pageY];
  ...
});
```

Reply:
normally yes, but in this case it's to supersede any/all other touch event handlers